package.json

```diff
@@ -25,23 +25,23 @@
     "babel-eslint": "^8.2.5",
     "babel-loader": "^7.1.2",
     "babel-plugin-transform-runtime": "^6.23.0",
-    "babel-preset-env": "~1.7.0",
+    "babel-preset-env": "~1.6.1",
```

Comment:
Why are so many dependencies downgraded?

Reply:
I think I ran into some issues while working on the hello app Friday so I downgraded to the previous versions, but it appears to be working now so I'll revert them.